python/lsst/ip/isr/isrFunctions.py

```diff
@@ -221,7 +221,9 @@ def growMasks(mask, radius=0, maskNameList=['BAD'], maskValue="BAD"):
 
 
 def maskITLEdgeBleed(ccdExposure, badAmpDict,
-                     fpCore, itlEdgeBleedThreshold=5000.,
+                     fpCore, itlEdgeBleedSatMinArea=10000,
+                     itlEdgeBleedSatMaxArea=100000,
+                     itlEdgeBleedThreshold=5000.,
```

Comment:
It's usually bad form to insert new keywords rather than put them at the end; it probably doesn't matter but I do want to flag that.